scripts/schmidt_comparison.py

```diff
@@ -0,0 +1,48 @@
+import sys
+sys.path.insert(0,'../chamber/models')
```

Comment:
Yes,there is an error importing using "from chamber.models import models as mod" or similar ways, I think this is because there are folders and .py files of the same name in the directory.